.circleci/config.yml

```diff
@@ -59,7 +59,7 @@ browsers_unit_tests: &browsers_unit_tests
         command: yarn codecov:browser
 
 jobs:
-  lint:
+  lint_&_docs:
```

Comment:
Can we make the `yarn install` only install dependencies required for linting and docs instead of all dependencies?

Reply:
Sure, I planned to do the in the next PR.